Documentation/block.md

```diff
@@ -117,6 +121,9 @@ To be able to use an erasure coded pool you need to create two pools (as seen be
 The replicated pool must be specified as the `blockPool` parameter. It is used for the metadata of the RBD images.
 The erasure coded pool must be set as the `dataBlockPool` parameter below. It is used for the data of the RBD images.
 
+**NOTE** This example requires you to have **at least 3 bluestore OSDs each on a different node**.
+This is because the below `erasureCoded` chunk settings require at least 3 bluestore OSDs and as [`failureDomain` setting](ceph-pool-crd.md#spec) to `host` (default), each OSD needs to be on a different nodes.
```

Comment:
- This one doesn't look like it has `failureDomain: host`. 
- For the EC examples what if you say "at least 3 bluestore OSDs". No need to mention independent nodes, right?

Reply:
1. I have added the missing `failureDomain: host`.
2. The default `failureDomain` is `host` so the independet nodes need to be mentioned.